app/templates/forms/user_profile.html

```diff
@@ -0,0 +1,29 @@
+<div class="col-6">
```

Comment:
Since this form will be included inside of the `templates/flask_user/edit_user_profile.html` template, we should name it to `edit_user_profile_form.html`.